docs/pages/setup/reference/audit.mdx

```diff
@@ -78,22 +78,73 @@ format:
+| db.create                     | A database has been added to Database Access. |
+| db.delete                     | A database has been deleted from Database Access. |
+| db.session.end                | Client has disconnected from a database. |
+| db.session.query              | Client has executed a query against a database. |
+| db.session.query.failed       | Client has executed a database query/command that was unsuccessful. |
+| db.session.start              | Client has connected to a database. |
+| db.update                     | A database has been updaed in Database Access. |
+| exec                          | Remote command has been executed via SSH, like `tsh ssh root@node ls /`. The following fields will be logged: `{"command": "ls /", "exitCode": 0, "exitError": ""}` |
+| github.created                | A [GitHub connector](https://goteleport.com/resources/guides/using-github-for-sso/) has been created. |
+| github.deleted                | A [GitHub connector](https://goteleport.com/resources/guides/using-github-for-sso/) has been deleted. |
+| kube.request                  | Client has issued a generic (non `kubectl exec`) Kubernetes Access request. |
+| lock.created                  | A [session or identity lock](https://goteleport.com/docs/access-controls/guides/locking/) has been created. |
+| lock.deleted                  | A [session or identity lock](https://goteleport.com/docs/access-controls/guides/locking/) has been deleted. |
+| mfa.add                       | A MFA device has been added to a user. |
+| mfa.delete                    | A MFA device has been removed from a user. |
+| oidc.created                  | A [OIDC connector](https://goteleport.com/docs/enterprise/sso/oidc/) has been created. |
+| oidc.deleted                  | A [OIDC connector](https://goteleport.com/docs/enterprise/sso/oidc/) has been deleted. |
+| port                          | Client has requested SSH port forwarding. |
+| privilege_token.create        | A user privilege token has been created. |
+| recovery_code.generated       | A recovery token has been generated. |
+| recovery_code.used            | A recovery token has been used. |
+| recovery_token.create         | A recovery code has been created.|
+| reset_password_token.create   | A password reset token has been created. |
+| resize                        | Terminal has been resized. |
+| role.created                  | A role has been added within Teleport. |
+| role.deleted                  | A role has been deleted within Teleport. |
+| saml.created                  | A [SAML connector](https://goteleport.com/docs/enterprise/sso/okta/) has been added. |
+| saml.deleted                  | A [SAML connector](https://goteleport.com/docs/enterprise/sso/okta/) has been deleted. |
+| scp                           | Remote file copy has been executed. The following fields will be logged: `{"path": "/path/to/file.txt", "len": 32344, "action": "read" }` |
```

Comment:
With other fields in this list, I think using a sample value to illustrate the field's type makes sense, but `action`'s value seems to be one of a constrained list—should we mention all of the the possible values? If all of the values are self explanatory, though, we can probably leave this as-is.